docs/installation/est/Installing_EST.md

```diff
@@ -48,91 +45,22 @@ Enabled profile "estServiceCert"
 --------------------------------
 ```
 
-
-EST Subsystem Installation
---------------------------
-Install the *EST subsystem* via dnf command.
+Finally, install the *EST subsystem* via dnf command.
 
 ```
 # dnf install dogtag-pki-est
 ```
 
-Create the *EST subsytem* inside the pki server instance:
-
-```
-# pki-server est-create
-```
-
-Configure the issuance backend. The class `org.dogtagpki.est.DogtagRABackend` is used for the *Dogtag CA*. This requires:
-
-- the **url** parameter pointing to the CA subsystem;
-- credentials of an RA account, **username** and **password**, that is authorised to issue certificates using the configured profile;
-  - is also possible to use TLS client certificate authentication to authenticate to the CA subsystem.
-- the **profile**.
 
+## EST Subsystem Installation
 
-```
-# cat >/var/lib/pki/pki-tomcat/conf/est/backend.conf <<EOF
-class=org.dogtagpki.est.DogtagRABackend
-url=https://$(hostname):8443
-profile=estServiceCert
-username=est-ra-1
-password=est4ever
-EOF
-```
+There are two options for the installation:
```

Comment:
Is there any advantage to pki-server that we want to continue providing it as an option?

Reply:
The pki-server tool provides more control over the installation process (i.e. fine-grained installation). If something fails, it can be fixed immediately, then the installation can resume from that point. With pkispawn it's all or nothing. pkispawn does many things at once so all params must be set correctly at the beginning. If something fails, usually you'd have to clean up the environment and start over from the beginning. This could be problematic if pkispawn leaves many orphan keys in HSM or starts replicating a huge database.

Reply:
Also for containers we don't want to do everything that pkispawn does, so pki-server tool provides a mechanism to do what's needed for containers.

I think for the primary EST doc we could suggest users to use pkispawn, but we should keep the EST doc using pki-server if they need more control over the installation process.

Reply:
@edewata Thanks for the clarification. Yes I think we should keep both options available, but perhaps add the explanation to help with choices?

Reply:
I was thinking the main EST install doc should just use pkispawn, but we also provide a link that says something like "Manual EST Installation" or "Advanced EST Installation" to another doc using pki-server. We can include the above explanation in that other doc. This way the user won't be forced to read, understand the differences, then make a choice before they can start the installation, but they can try pkispawn with the sample config files first (which is what most people want to do initially), then if it's not enough they can try pki-server. What do you think?